src/gpu/intel/softmax/xe.hpp

```diff
@@ -297,6 +302,9 @@ struct xe_bwd_t : public primitive_t {
                     != format_tag::undef);
             is_blk = (diff_src_d.matches_one_of_tag(nCw16c, nChw16c, nCdhw16c)
                     != format_tag::undef);
+            VDISPATCH_SOFTMAX(
+                    (is_blk && subgroup_size == 16) || diff_src_d.is_plain(),
+                    VERBOSE_UNSUPPORTED_TAG);
```

Comment:
Instead of choosing another implementation when `is_blk` and subgroup_size is too large, could we instead just "snap" the subgroup size to 16 if it's in one of these formats?